enhancements/sig-policy/128-resource-selector/README.md

```diff
@@ -0,0 +1,142 @@
+# Resource selector for `ConfigurationPolicy`
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Add a resource selector to pair with the existing `namespaceSelector` in order to allow a
+`ConfigurationPolicy` to filter and select objects when an `objectDefinition` doesn't have a name
+provided. The resource selector would also enable the enforcement of `objectDefinitions` without a
+name.
+
+## Motivation
+
+The `ConfigurationPolicy` currently has a `namespaceSelector` to configure the controller to filter
+and range over namespaces relevant to a particular namespaced `objectDefinition`. Furthermore,
+currently an `objectDefinition` without a name is feasible but is not enforceable. A
+`resourceSelector` would both pair well with the `namespaceSelector` for further object filtering as
+well as enable an enforceable unnamed `objectDefinition` by providing the controller with a list of
+names to handle.
+
+Adding a `resourceSelector` would lower the bar to entry for `ConfigurationPolicy`, adding an
+additional configuration available to users without requiring knowledge of Go templating as is
+required with the alternative, `object-templates-raw`.
+
+### Goals
+
+- Add a `resourceSelector` to `ConfigurationPolicy` to allow the controller to handle objects
```

Comment:
Do you think another goal is to consolidate the compliance message for an object-template with a resource selector? One of the main drawbacks with object-templates-raw is you get a separate compliance message per object-template. The use of a namespaceSelector currently consolidates the compliance message but that is limited as you mention in the introduction.

Reply:
That's a good point--the message could probably be refactored to list out the objects per namespace rather than have a message for every object--is that what you mean?

Reply:
Either that or consolidate to something like `Pods found as specified: namespace1/name1, namespace2/name2; Pods updated: namespace1/name3 ` per object-template with a resource selector.